composer.json

```diff
@@ -20,13 +20,16 @@
         }
     ],
     "require": {
-        "php": "^7.3",
-        "illuminate/http": "^6.0|^7.0|^8.0"
+        "php": "^7.3|^8.0",
+        "illuminate/http": "^6.0|^7.0|^8.0",
+        "laravel/framework": "6.*",
```

Comment:
I haven't tested this locally yet, but won't this lock the package to Laravel 6 only?

This "works" in Github Actions because I am explicitly installing different versions of Laravel and Symfony based on the matrix (see https://github.com/ArondeParon/laravel-request-sanitizer/blob/main/.github/workflows/run-tests.yml#L44) but I think this will break in production.